<?xml version="1.0"?>
<launch>
  <node pkg="pointcloud_to_laserscan" type="pointcloud_to_laserscan_node" name="pointcloud_to_laserscan_node">
	  <remap from="cloud_in" to="velodyne_obstacles"/>
    <param name="min_height" type="double" value="-1.5"/>
    <param name="max_height" type="double" value="1.5"/>
    <param name="angle_min" type="double" value="-1.5708"/>
    <param name="angle_max" type="double" value="1.5708"/>
    <param name="range_min" type="double" value="0.1"/>
    <param name="range_max" type="double" value="10.0"/>
    <param name="angle_increment" type="double" value="0.00436471"/>
    <param name="target_frame" type="str" value="base_link"/>

  </node>
</launch>
